main.tf

```diff
@@ -23,9 +23,11 @@ resource "aws_iam_role_policy" "lambda_additional_policy" {
   count  = "${length(var.iam_policy_document) > 0 ? 1 : 0}"
 }
 
-resource "random_id" "randomiser" {
-  byte_length = 8
-  prefix      = "${var.product_domain}-${var.lambda_name}-"
+module "randomiser" {
+  source = "github.com/traveloka/terraform-aws-resource-naming.git?ref=v0.6.0"
+
+  name_prefix   = "${var.product_domain}-${var.lambda_name}"
```

Comment:
This needs to be product_domain|service_name.

Reply:
Hi @oktaviandi-nugraha I put `product_domain` in there because there is no `service_name` on `variables.tf`. I did not create the variable because I thought this module does not need it, as you originally create the module without that variable. 

But it is more make sense if Lambda that is going to be created using the module is thought as a service, because yes it is a service. So it will need `service_name` as variable, and this line will be changed to 
```hcl
  name_prefix   = "${var.service_name}-${var.lambda_name}"
```
what do you think?